src/coreclr/tools/aot/ILCompiler.Build.Tasks/ILCompiler.Build.Tasks.csproj

```diff
@@ -9,12 +9,16 @@
     <OutputPath>$(RuntimeBinDir)/ilc-published/netstandard</OutputPath>
     <Configurations>Debug;Release;Checked</Configurations>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <!--
+      The SDK warns if a project targets a framework that doesn't support trimming and IsTrimmable is true.
+      We set IsTrimmable higher up in the Directory.Build.props files, so we reset it to false here.
+    -->
+    <IsTrimmable>false</IsTrimmable>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" />
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
   </ItemGroup>
-
```

Comment:
Please undo this deletion. We aim towards having our project files represent the `dotnet new` ones which include a new line after the project start tag and before the project close tag.